packages/vuetify/src/components/VBtn/VBtn.cy.spec.tsx

```diff
@@ -1,28 +1,283 @@
+        .should('not.have.class', 'bg-success')
+    })
+  })
+
+  describe.skip('activeClass', () => {
+    it('should use custom active-class', () => {
+      cy.mount(<VBtn activeClass="my-active-class">Active Class</VBtn>)
+        .get('.my-active-class')
+        .should('exist')
+    })
+  })
+
+  describe('retainFocusOnClick', () => {
+    it('when true, the button has focus', () => {
+      cy.mount(<VBtn retainFocusOnClick>My button</VBtn>)
+        .get('button')
+        .click()
+        .should('have.focus')
+    })
+
+    it.skip('when false, the button does not capture focus', () => {
+      cy.mount(<VBtn retainFocusOnClick={ false }>My button</VBtn>)
+        .get('button')
+        .click()
+        .should('not.have.focus')
+    })
+  })
+
+  describe.skip('loading', () => {
+    it('when using the loader slot, do not show the progress indicator', () => {
+      cy.mount(() => (
+        <VBtn loading v-slots={ { loader: () => <span>{ loadingText }</span> } } />
+      ))
+        .get('button')
+        .should('contain.text', loadingText)
+        .get('role[progressbar]')
+        .should('not.exist')
+    })
+
+    // custom loaders are not yet implemented
+    it('when loading is true, show the progress indicator', () => {
+      cy.mount(<VBtn loading>{ loadingText }</VBtn>)
+        .get('button')
+        .should('contain.text', loadingText)
+        .get('role[progressbar]')
+        .should('be.visible')
+    })
+  })
+
+  // v-btn--tile isn't implemented at all
+  describe.skip('tile', () => {
+    it('applies the tile class when true', () => {
+      cy.mount(<VBtn tile />)
+        .get('button')
+        .should('contain.class', 'v-btn--tile')
+    })
+
+    it('does not apply the tile class when false', () => {
+      cy.mount(<VBtn tile={ false } />)
+        .get('button')
+        .should('not.contain.class', 'v-btn--tile')
+    })
+  })
+
+  describe.skip('href', () => {
+    it('should render an <a> tag when using href prop', () => {
+      cy.mount(<VBtn href={ anchor.href }>Click me</VBtn>)
+        .get('button')
+        .click()
+        .get('a') // currently not rendering the <a> tag at all
+        .should('contain.text', 'Click me')
+        .should('have.focus')
+        .hash()
+        .should('contain', anchor.hash)
+    })
+  })
+
+  describe.skip('value', () => {
+    // none of the "value" props are implemented yet
+    it('should stringify non string|number values', () => {
+      const objectValue = { value: { hello: 'world' } }
+      const numberValue = { value: 2 }
+
+      cy.mount(<VBtn value={ objectValue }></VBtn>)
+        .get('button')
+        .should('contain.text', JSON.stringify(objectValue, null, 0))
+        .mount(<VBtn value={ numberValue } />)
+        .get('button')
+        .should('contain.text', numberValue.value)
+    })
+  })
+
+  // I think there's an issue with setProps,
+  // _or_ the prod code isn't working
+  describe.skip('Reactivity', () => {
```

Comment:
@lmiller1990 can you look into the setProps weirdness?

Reply:
I can do it this weekend